src/components/ApplicationGroup/AppGroup.service.ts

```diff
@@ -189,27 +191,17 @@ export const getConfigAppList = (envId: number, appIds: string): Promise<ConfigA
     return get(`${Routes.ENVIRONMENT}/${envId}/${Routes.ENV_APPLICATIONS}${getFilteredAppQueryString(appIds)}`)
 }
 
-export const getEnvAppList = (
-    params?: {
-        envName?: string
-        clusterIds?: string
-        offset?: string
-        size?: string
-    },
-    signal?: AbortSignal,
-): Promise<EnvAppType> => {
+export const getEnvAppList = (filterConfig?: Partial<AppGroupFilterConfig>, signal?: AbortSignal): Promise<EnvAppType> => {
     const options = signal ? { signal } : null
+    const {searchKey, cluster, offset, pageSize} = filterConfig
 
-    if (params) {
-        const urlParams = Object.entries(params).map(([key, value]) => {
-            if (!value) {
-                return
-            }
-            return `${key}=${value}`
-        })
-        return get(`${Routes.ENVIRONMENT_APPS}?${urlParams.filter((s) => s).join('&')}`, options)
-    }
-    return get(Routes.ENVIRONMENT_APPS, options)
+    const url = getUrlWithSearchParams(Routes.ENVIRONMENT_APPS, {
+        envName: searchKey,
+        clusterIds: cluster?.join(),
```

Comment:
Suggestion: There is support for array inside getUrlWithSearchParams please check if we can leverage that?

Reply:
we are sending cluster ids as comma seperated values here